doc/source/walkthrough.rst

```diff
@@ -168,6 +168,20 @@ Further, remote functions can return multiple object IDs.
 
   a_id, b_id, c_id = return_multiple.remote()
 
+Remote functions can be canceled by calling ``ray.cancel`` on the returned Object ID. 
```

Comment:
Can you include some of the semantics here? Is this idempotent? What if a task is partially complete? Can I do it on a set of tasks? etc.